src/handlers/state.ts

```diff
@@ -0,0 +1,5 @@
+(<any>window).state = {
+    activeChatId: -1,
+    newMessageText: "",
+    dataChangeMode: false
+};
```

Comment:
```suggestion
};

```
Очень распространённая рекомендация вставлять пустую строку в конце файла. Чтобы не делать этого самому, можно настроить editorconfig файл в корне проекта, тогда редакторы кода будут сами вставлять в конце файлов пустую строку.

Подробнее о пустой строке можно прочитать тут.
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log